rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -2480,6 +2481,30 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
      * evaluate the options in the input argString, and if any Options defined for the Job have required=true, have a
      * defaultValue, and have null value in the input properties, then append the default option value to the argString
      */
+    def Map addRemoteOptionSelected(ScheduledExecution scheduledExecution, Map optparams, UserAndRolesAuthContext authContext = null) throws ExecutionServiceException {
```

Comment:
the javadoc for the wrong method is now attached to this method, you should make sure the javadoc stays with the correct method, and add new javadoc for this method